e2e_tests/database/test_database.json

```diff
@@ -389,6 +389,32 @@
     "change_message": "[{\"changed\": {\"fields\": [\"Bewertung Klimaaktionsplan\", \"Bewertung Umsetzungsstand\"]}}]"
   }
 },
+{
```

Comment:
These entries can be removed from the DB to keep the file from growing. Not important for this PR, though.